lib/netconf/local/genesis.json

```diff
@@ -175,8 +175,10 @@
       "params": {
         "max_withdrawal_per_block": 32,
         "max_sweep_per_block": 128,
-        "min_partial_withdrawal_amount": 100000000
-      }
+        "min_partial_withdrawal_amount": 100000000,
+        "epoch_identifier": "minute"
```

Comment:
How do we add epoch duration in genesis

Reply:
Only epoch identifier is needed in evmstaking module. Epoch info such as epoch duration is in `epochs` module, and it can be added in genesis.json for now. There is no way to add epoch info as cosmos-sdk do.

Reply:
It will be great to add configuration in genesis.json to change the number of blocks per epoch. It helps to perform tests in reasonable amount of time.

Reply:
The epoch duration can be set in `epochs` module. For local or mininet test, we can use shorter epoch by setting shorter epoch. `emvstaking` module depends on `epochs` module

Reply:
If we can set the duration in genesis.json, it will make setting different networks easier. Otherwise, different networks need to manage different binaries.

Reply:
I think it's ok if each network has different genesis.json file (to be exact, different epoch identifier for epoch staking).

If we add duration in `evmstaking` param (and add to genesis.json), we don't need `epochs` module in `evmstaking` module and we don't need `epochs` module in Story for now. `evmstaking` module can be independent with `epochs` module. The reason why I have added and used `epochs` module for epoch staking is that I wanna make `epochs` module manage epoch stuffs in our app for modularity. Do you think that it better than current design that `evmstaking` module have independent epoch info for epoch staking?

Reply:
Yeah I think we should add epoch info we use for evmstaking to epochs module's genesis file. I guess what you meant is that it should be in another PR?

Reply:
Done in https://github.com/piplabs/story/pull/181